galleries/examples/color/individual_colors_from_cmap.py

```diff
@@ -0,0 +1,61 @@
+"""
+===========================================
+Selecting individual colors from a colormap
+===========================================
+
+Sometimes we want to use more colors or a different set of colors than the default color
+cycle provides. Selecting individual colors from one of the provided colormaps can be a
+convenient way to do this.
+
+Once we have hold of a `.Colormap` instance, the individual colors can be accessed
+by passing it an index.  If we want a specific number of colors taken at regular
+intervals from a continuous colormap, we can create a new colormap using the
+`~.Colormap.resampled` method.
+
+For more details about manipulating colormaps, see :ref:`colormap-manipulation`.
+"""
+
+import matplotlib.pyplot as plt
+
+import matplotlib as mpl
+
+n_lines = 21
+
+cmap = mpl.colormaps.get_cmap('plasma').resampled(n_lines)
+
+fig, ax = plt.subplots(layout='constrained')
+
+for i in range(n_lines):
+    ax.plot([0, i], color=cmap(i))
+
+plt.show()
+
+# %%
+# Alternatively, we may want to replace the default color cycle with a different
+# discrete set of colors.  We can use a `~cycler.cycler` instance within `.rcParams` to
```

Comment:
Sorry to dribble comments, but I found "Alternatively" poor guidance here, as I immediately thought it would be "alternative" was of getting the colors, whereas its an alternative way of passing the colors to `plot`.  

Maybe something like: 
```suggestion
#  Instead of specifying the colors one by one in ``plot``, we can create a temporary `~cycler.cycler` that has the list of colors specified.  
```